### ISSUE TITLE
Fencing Grace Broken

### ISSUE THREAD

**Issue by justanotherusername2344** [Vek17/TabletopTweaks-Base#442]
With this mod enabled Fencing Grace does nothing, with it disabled it works appropriately by applying dexterity bonuses to the selected weapon.  I tried disabling the fencing grace feat fix and nothing changed only disabling this mod allowed fencing grace to work again

**Comment by Vek17** [comment]
I need more information, Fencing grace works correctly in my test saves and disabling it should result in no changed made. 

What weapon are you using, what fencing grace version, what other mods are using, what is in your off hand, are you two handing your weapon (do you have Use Weapon One Handed Toggled on).

**Comment by justanotherusername2344** [comment]
I did not have use weapon in one hand toggled on. I didn't notice that button. that fixed my issue, thanks. 

I'm using a rapier, it's the same with all rapiers. I tested by disabling every mod except this one and my character loses the dexterity bonus with this mod enabled but gains a strength bonus. I selected Fencing Grace Rapier on level up but in my character sheet it just lists it as fencing grace. Nothing in off hand.
